docs/DEVELOPMENT.md

```diff
@@ -92,22 +92,35 @@ In order to be merged your opened pull request (PR) should pass the static analy
 
 The e2e tests jobs are executed on a variety of CcRuntime, configurations and platforms. These jobs that require confidential hardware (Intel TDX, AMD SEV, IBM SE, etc...) run on bare-metal machines and are often referred as "TEE tests". The remaining tests (a.k.a "Non-TEE") are executed on Virtual Machines (VM) deployed on-demand.
 
+This is a list of the bare-metal machines and VMs that are utilized by the project for CI testing:
+
+| Machine               | Type       | TEE Capability | Assigned Labels                          |
+| ---                   | ---        | ---            | ---                                      |
+| az-ubuntu-2004        | virtual    | Non-TEE        |                                          |
+| az-ubuntu-2204        | virtual    | Non-TEE        |                                          |
+| s390x-runner-01       | virtual    | Non-TEE        | S390X                                    |
```

Comment:
For the runner `s390x-runner-01`, `s390x` and `s390x-large` are used. There is another one named ` s390x-runner-02` whose label is only `s390x`.

Reply:
So the final status would look like:

| s390x-runner-01 | virtual | Non-TEE | s390x, s390x-large
| s390x-runner-02 | virtual | Non-TEE | s390x

Reply:
Done.